python/cac_tripplanner/destinations/views.py

```diff
@@ -47,10 +47,10 @@ def home(request):
     }
     if request.GET.get('destination') is not None:
         # If there's a destination in the URL, go right to directions
-        context['tab'] = 'map'
+        context['body_class'] = 'body-map body-map-dir body-has-sidebar-banner'
```

Comment:
The Python template context is not a good place to put classes, which will change frequently and which we will expect to find set via JavaScript manipulation and not go looking here. I think it might be good to take the class out of the base template altogether.